api/db-cdr/generate-cdr/make-bq-data-dump.sh

```diff
@@ -40,14 +40,22 @@ fi
 
 echo "Dumping tables to csv from $BUCKET\n"
 
+##echo "Echoing dataset name"
+##echo ${DATASET}
+
 # Get tables in project, stripping out tableId.
 # Note tables larger than 1 G need to be dumped into more than one file.
 # concept_relationship and concept are only big ones now.
-tables=(achilles_analysis achilles_results achilles_results_concept concept concept_relationship criteria db_domain domain vocabulary )
+if [[ $DATASET == *public* ]] || [[ $DATASET == *PUBLIC* ]];
+then
+    tables=(achilles_analysis achilles_results achilles_results_concept concept concept_relationship criteria db_domain domain vocabulary)
+else
+    tables=(achilles_analysis achilles_results achilles_results_concept concept concept_relationship criteria db_domain domain vocabulary concept_ancestor)
+fi
 
 for table in ${tables[@]}; do
   echo "Dumping table : $table"
-  if [[ $table =~ ^(concept|concept_relationship)$ ]]
+  if [[ $table =~ ^(concept|concept_relationship|concept_ancestor|achilles_results)$ ]]
```

Comment:
achilles_results doesn't need to be in here because the data is less than 1g.

Reply:
My mistake. I had to add it when my query was generating really large number of rows because of not excluding some questions.

Reply:
Done.